src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -676,7 +677,7 @@ namespace winrt::TerminalApp::implementation
                 }
                 tab->_focusState = WUX::FocusState::Programmatic;
                 // This tab has gained focus, remove the bell indicator if it is active
-                if (tab->_headerControl.BellIndicator())
+                if (tab->TabStatus().BellIndicator())

```

Comment:
Curious, this one uses `TabStatus()` but the others use `_tabStatus`. I'm assumuing there's not _really_ a reason, right? (I don't mind, I guess I'm just curious if there _is_ a reason)

Reply:
No reason, just lack of focus. I will fix it. :blush: